### PR TITLE
LPS-128811 Adjust info button size and spacing between elements

### PR DESCRIPTION
**Pull request by ramonmsp**

### CODE REVIEW

File: modules/apps/portal-workflow/portal-workflow-web/src/main/resources/META-INF/resources/css/_definition.scss [hunk 1]
```diff
@@ -73,8 +86,12 @@
 }
 
 .lfr-sidebar-list-group-workflow {
+	// &.container-view {
+	// 	padding-top: 0px;
+	// }
```
Comment (interaminense) [inline modules/apps/portal-workflow/portal-workflow-web/src/main/resources/META-INF/resources/css/_definition.scss:91]: Hey @ramonmsp, could you remove this comment?

**Review by interaminense — CHANGES_REQUESTED** [review]

**Comment by interaminense** [comment]
Hey @ramonmsp, I think you can reduce the last commits in just one commit. I can see that with you if you want.

**Review by interaminense — APPROVED** [review]

**Review by naraandrade — APPROVED** [review]

**Comment by naraandrade** [comment]
ci:forward

**Comment by naraandrade** [comment]
ci:forward

**Pull request closed without merging**